CODING.md

```diff
@@ -222,6 +222,15 @@ The CI tooling will check and enforce the order and required sections when a spr
   * When no golang code is involved, the context prefix before the colon can refer to directories or top-level files instead.
     * `build-aux,.github/workflows: limit make processes with nproc`
 
+* Merging
+  * Only use `Squash and Merge` or `Rebase and Merge`, never `Create a merge commit`
```

Comment:
can we enable this in the repository settings then? This should be in the settings, under `Allow merge commits`

Reply:
I like this idea, changed it. Wish there was a way set squash as default now, its rebase unfortunately.